### ISSUE TITLE
web view height issue

### ISSUE THREAD

**Issue by mcebular** [TheFinestArtist/FinestWebView-Android#110]
Still having issue #14 .. Tried visiting duckduckgo.com, same issue tested on Marshmallow and Kitkat versions. There a fix for it?
![screenshot](https://cloud.githubusercontent.com/assets/17457833/21518551/a07c92c2-cce7-11e6-8c0e-85bcedb087f9.png)

**Comment by NoelChew** [comment]
Doesn't work on Android 7.0 too.

**Comment by amitsri20** [comment]
Yeah..i am using version 1.2.7 and facing the same issue..as mentioned in #14 where to set minHeight for webview?

**Comment by sckoh** [comment]
i have the same problem too

**Comment by ponnamkarthik** [comment]
> [https://github.com/PonnamKarthik/FinestWebView-Android](https://github.com/PonnamKarthik/FinestWebView-Android)

Check the code from above link
webview height issue is fixed

**Comment by makunomark** [comment]
@PonnamKarthik This link seems broken

**Comment by ponnamkarthik** [comment]
@makunomark  
> [Correct Link](https://github.com/PonnamKarthik/FinestWebView-Android)
